VisualStudio.gitignore

```diff
@@ -1,16 +1,18 @@
-## Ignore Visual Studio temporary files, build results, and
-## files generated by popular Visual Studio add-ons.
-
-# User-specific files
+# temporary files generated by Visual Studio specific to the local user
+# contains profile information and other metadata
+# remove: if no longer targeting Visual Studio 2013 or earlier
 *.suo
 *.user
 *.userosscache
 *.sln.docstates
-
-# User-specific files (MonoDevelop/Xamarin Studio)
+#
+# new location for Visual Studio 2015
```

Comment:
This is an example of a rule which most users don't currently need (this version of Visual Studio is in RC) but having it enabled currently is good for future-proofing things. As such, it doesn't have an `include` field defined.